lib/controller/database_controller.dart

```diff
@@ -30,7 +31,7 @@ class DatabaseController extends GetxController {
       },
       "upi": "9876543210@upi",
       "type": "veg",
-      "loc": const GeoPoint(30.2829736, 76.7105632),
+      "loc":restaurantlocation==""? const GeoPoint(30.2829736, 76.7105632):restaurantlocation,
```

Comment:
why are you fixing the location, make it dynamic

Reply:
It is a default value, reason behind is whenever any error occurs while fetching the value from gps location then in default case it will consider it.

Reply:
It is a default value, reason behind is whenever any error occurs while fetching the value from gps location then in default case it will consider it.